test/test_batch_consistency.py

```diff
@@ -84,7 +84,8 @@ def test_griffinlim(self):
         length = 1000
         tensor = torch.rand((1, 201, 6))
         _test_batch(
-            F.griffinlim, tensor, window, n_fft, hop, ws, power, normalize, n_iter, momentum, length, 0, atol=5e-5
+            F.griffinlim, tensor, window, n_fft, hop, ws, power, normalize, n_iter, momentum, length, 0,
+            atol=5e-5, rtol=1e-5,
```

Comment:
why is the tolerance changed?

Reply:
To use `assert_allclose`, you have to either 1. give both `atol` and `rtol` or 2. give neither of them. So I filled the values which was used originally.

Reply:
sounds good!